### DISCUSSION TITLE
I'm asking for help with graphing njsPC data.

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#474] [App and O/S Config Questions] [answered]
I tried using nodejs-poolController-graph, but get an error upon npm start.
https://github.com/wurmr/nodejs-poolController-graph
from @wurmr 

> nodejs-pool-controller-graph@1.0.0 start /home/pi/nodejs-poolController-graph
> node dist/index.js

internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '/home/pi/nodejs-poolController-graph/dist/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nodejs-pool-controller-graph@1.0.0 start: `node dist/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nodejs-pool-controller-graph@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Comment by rstrouse** [comment, marked as answer]
I use InfluxDb and Grafana.  Maybe @wurmr can help you with that module.

**Reply by tagyoureit** [reply]
That module is a GraphQL wrapper for the API - not a graphing solution.

**Reply by raymop** [reply]
I found some good references for setting up influx and grafana.
